.github/workflows/actions.yml

```diff
@@ -54,7 +54,7 @@ jobs:
             -e ./benchadapt/python \
             -e ./benchconnect \
             -e ./benchrun/python \
-            -e .
+            -e ./legacy
```

Comment:
Can you make this same change here?
https://github.com/conbench/conbench/blob/42ef379825e0bb8d7cedc00f0d143e84f102c60b/.github/workflows/publish_docs.yml#L26